packages/components/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,62 @@
+import { styled } from "../../stitches.config";
+import Wrapper from "../Wrapper";
+import React from "react";
+import Heading from "../Heading";
+
+export default function Hero({
+  heading,
+  ImageComponent,
+}: {
+  heading: React.ReactNode;
+  ImageComponent: (
+    props: React.ImgHTMLAttributes<HTMLImageElement>
+  ) => React.ReactElement;
+}) {
+  return (
+    <Container>
+      <Wrapper>
+        <Content>
+          <Heading>{heading}</Heading>
+          <Frame>
+            <ImageComponent width={984} height={464} />
```

Comment:
Using an `ImageComponent` input here so that there isn't a `next` dependency in this library. Alternatively, we could have a next dependency and then accept src/alt as props